completions/cmux.bash

```
@@ -0,0 +1,305 @@
+    }
+    # Locate the command word (first non-option after `cmux`).
+    local i cmd=""
+    for ((i=1; i < COMP_CWORD; i++)); do
+        case "${COMP_WORDS[i]}" in
+            -*) ;;
+            *) cmd="${COMP_WORDS[i]}"; break ;;
+        esac
+    done
+    local commands="agent-hibernation auth bind-key break-pane browser browser-back browser-forward browser-reload browser-status capabilities capture-pane claude-teams clear-history clear-log clear-notifications clear-progress clear-status close-surface close-window close-workspace cloud codex codex-teams config copy-mode current-window current-workspace debug-terminals detach-tab diff disable-browser dismiss-notification display-message docs drag-surface-to-split enable-browser events feed feedback find-window focus-pane focus-panel focus-webview focus-window get-url help hooks identify is-webview-focused join-pane jump-to-unread last-pane last-window list-buffers list-log list-notifications list-pane-surfaces list-panels list-panes list-status list-windows list-workspaces log login logout mark-notification-read markdown memory mobile move-surface move-tab-to-new-workspace move-workspace-to-window navigate new-pane new-split new-surface new-window new-workspace next-window notify omc omo omx open open-browser open-notification paste-buffer ping pipe-pane popup previous-window read-screen refresh-surfaces reload-config remote-daemon-status rename-tab rename-window rename-workspace reorder-surface reorder-workspace reorder-workspaces resize-pane respawn-pane restore-session right-sidebar rpc select-workspace send send-key send-key-panel send-panel set-app-focus set-buffer set-progress set-status settings shortcuts sidebar sidebar-state simulate-app-active split-off ssh ssh-session-attach ssh-session-cleanup ssh-session-list ssh-tmux surface surface-health surface-resume swap-pane tab-action themes top tree trigger-flash unbind-key version vm wait-for welcome workspace workspace-action workspace-group"
+    if [[ -z $cmd ]]; then
+        COMPREPLY=( $(compgen -W "$commands" -- "$cur") )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)